spec/enumerable_spec.rb

```diff
@@ -0,0 +1,244 @@
+require_relative '../enumerable'
+
+describe 'enumerable' do
+  describe '#my_each' do
+    let(:item) { [1, 2, 3, 4] }
```

Comment:
- Kindly move this ```let(:item) { [1, 2, 3, 4] }``` line and place it between lines 3 and 4 thus making it a global variable. Then, remove it from all other places they currently exist.